app/models/key_pair.rb

```diff
@@ -0,0 +1,44 @@
+# RSA key pair used for server-side signing
+class KeyPair < ActiveRecord::Base
+  SECRET_KEY = ENV.fetch('ENCRYPTED_PRIVATE_KEY_SECRET')
```

Comment:
this key uses any special format, or it's just an arbitrary string? 
We can add a default (as documentation, and for dev) on the `.env` file.